File: app/src/main/java/me/kirimin/mitsumine/ui/fragment/ReadLaterFeedFragment.java
package me.kirimin.mitsumine.ui.fragment;

import me.kirimin.mitsumine.db.FeedDAO;
import me.kirimin.mitsumine.model.Feed;

public class ReadLaterFeedFragment extends AbstractFeedFragment {

    @Override
    void requestFeed() {
        setFeed(FeedDAO.findByType(Feed.TYPE_READ_LATER));
        dismissRefreshing();
    }

    @Override
    boolean isUseReadLater() {
        return false;
    }

    @Override
    boolean isUseRead() {
        return true;
    }
}

File: app/src/main/java/me/kirimin/mitsumine/ui/fragment/ReadLaterFeedFragment.kt
package me.kirimin.mitsumine.ui.fragment

import me.kirimin.mitsumine.db.FeedDAO
import me.kirimin.mitsumine.model.Feed

public class ReadLaterFeedFragment : AbstractFeedFragment() {

    override fun requestFeed() {
        setFeed(FeedDAO.findByType(Feed.TYPE_READ_LATER))
        dismissRefreshing()
    }

    override fun isUseReadLater(): Boolean {
        return false
    }

    override fun isUseRead(): Boolean {
        return true
    }
}
